virtualization/windowscontainers/management/container_networking.md

```diff
@@ -124,7 +124,7 @@ Stop-Service docker
 The configuration file can be found at `c:\programdata\docker\runDockerDaemon.cmd`. Edit the following line, replacing `Virtual Switch` with the name of the virtual switch to be used by the Docker service.
 
 ```powershell
-docker daemon -D -b “New Switch Name"
+docker daemon -D -b "New Switch Name"
```

Comment:
Just CMD+SHIFT+F (or CTRL+SHIFT+F) with the `“` to search through the whole repo. Should be done for `”` as well.